e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubEdgeBaseDeployment.cs

```diff
@@ -32,8 +32,8 @@ public sealed class IoTHubEdgeBaseDeployment : DeploymentConfiguration {
 
         /// <inheritdoc />
         protected override IDictionary<string, IDictionary<string, object>> CreateDeploymentModules() {
-            var server = string.IsNullOrEmpty(_context.ContainerRegistryConfig.ContainerRegistryServer) ?
-                TestConstants.MicrosoftContainerRegistry : _context.ContainerRegistryConfig.ContainerRegistryServer;
+            // We should always consume edgeAgent and edgeHub from mcr.microsoft.com.
```

Comment:
Not for this PR, but for future, consider to add a variable (especially from pipeline) to input private bits of edgehub/edgeagent from a registry for testing/validating.